### DISCUSSION TITLE
Latex: Possiblity to remove 'Appendices' part title in table of contents

### DISCUSSION THREAD

**Discussion by fitzchivalrik** [quarto-dev/quarto-cli#1086] [Feature Requests]
I am in the process of porting my latex template for my thesis to quarto, using the `book` template.
When using `appendices` in the `book` yaml, quarto inserts `\addcontentsline{toc}{part}{Appendices}` right after `\appendix`.
Unfortunately, this does not adhere to the style guide I need to use. It would be great if this was configurable (`appendix-style: none` does not seem to affect this).
Related, I think the produced hyperlink is incorrect, as clicking on `Appendices` in the toc in the PDF moves to the section before the first appendix, however, I am unsure if this is not an error in my setup.

```yaml
book:
  chapters:
    - index.qmd
  appendices: # <-
    - appendix/first.qmd
```
https://github.com/quarto-dev/quarto-cli/blob/5a889b6316be4abcde8b0c3c5d4e4c2f58e4bbd4/src/resources/filters/quarto-pre/book-numbering.lua#L26-L32

Thanks for the software and your work, having one code base to produce a beautiful book-website _and_ PDF is really great.

**Comment by jjallaire** [comment]
Thanks for the feedback. I'm going to transfer this to our GitHub Issues for better tracking (we are currently closing down development for v1.0 so it may be a bit before we take this up)

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/1096
